golang/inbound.go

```diff
@@ -91,7 +90,12 @@ func (c *Connection) handleCallReq(frame *Frame) bool {
 	response.messageForFragment = func(initial bool) message {
 		if initial {
 			call.AddAnnotation(AnnotationKeyServerSend)
-			call.Report(callReq.Tracing, c.traceReporter)
+			targetEndpoint := TargetEndpoint{
```

Comment:
I just mean that in this diff, you are declaring `targetEndpoint` in 2 places:
1. In this file, right before you call `call.Report` which is inside of `messageForFragment` function.
2. In the `outbound.go` file, you do it outside of `messageForFragment`.

The logic of those 2 functions is similar, and they should look similar, and the variable should be declared in a consistent location. For now, I would suggest that `targetEndpoint` should be created right before you do `call.Report`.